test/e2e/parallel/space_cleanup_test.go

```diff
@@ -27,7 +27,7 @@ func TestSpaceAndSpaceBindingCleanup(t *testing.T) {
 	t.Run("for SpaceBinding", func(t *testing.T) {
 		t.Run("when space is deleted", func(t *testing.T) {
 			// given
-			space, _, spaceBinding := setupForSpaceBindingCleanupTest(t, awaitilities, memberAwait, "joe", "redhat")
+			space, _, spaceBinding := setupForSpaceBindingCleanupTest(t, awaitilities, memberAwait, "joe", "for-redhat")
```

Comment:
we should not use `redhat` as the name (or just a prefix) of the namespaces. It shouldn't be a big deal in OCP, but let's try to avoid it also here